tests/integration/basics.rs

```diff
@@ -40,3 +39,35 @@ authors = ["{{authors}}"]
             .contains("foobar-project")
     );
 }
+
+#[test]
+fn it_substitutes_crate_name_in_a_rust_file() {
+    let template = dir("template")
+        .file(
+            "main.rs",
+            r#"
+extern crate {{crate_name}};          
+"#,
+        )
+        .init_git()
+        .build();
+
+    let dir = dir("main").build();
+
+    Command::main_binary()
+        .unwrap()
+        .arg("generate")
+        .arg("--git")
+        .arg(template.path())
+        .arg("--name")
+        .arg("foobar-project")
+        .current_dir(&dir.path())
+        .assert()
+        .success()
+        .stdout(predicates::str::contains("Done!").from_utf8());
+
+    assert!(
+        dir.read("foobar-project/main.rs")
+            .contains("foobar_project")
```

Comment:
maybe assert that it also *doesn't* contain "foobar-project"?

Reply:
oh good call

Reply:
(i did that manually, so i don't know why i didn't think to add it. thank u!)